ansible/roles/common_installs/tasks/main.yml

```diff
@@ -32,7 +36,7 @@
     - htop
     - tmpreaper
     - mailutils
-    - npm
+    - nodejs
```

Comment:
this caused me quite a bit of pain. we have different installs of node on different machines which made the path to `bower` different. i think this should be the right way; i know we've gone back and forth on nodejs and npm many times. with this ppa nodejs should install npm as well. see: http://stackoverflow.com/a/16303380/835696

@dannyroberts @TylerSheffels